corgi/tasks/monitoring.py

```diff
@@ -57,8 +57,9 @@ def upsert_interval_task(module, task, hours=None, minutes=None):
     # task at all: https://github.com/celery/django-celery-beat/issues/221
     upsert_cron_task("errata_tool", "load_et_products", hour=0, minute=0)
     upsert_cron_task("prod_defs", "update_products", hour=1, minute=0)
-    upsert_cron_task("pulp", "update_cdn_repo_channels", hour=1, minute=30)
-    upsert_cron_task("rhel_compose", "save_composes", hour=3, minute=0)
+    upsert_cron_task("pulp", "update_cdn_repo_channels", hour=2, minute=0)
+    upsert_cron_task("rhel_compose", "get_builds", hour=3, minute=0)
+    upsert_cron_task("rhel_compose", "save_composes", hour=4, minute=0)
```

Comment:
I think we should probably update this task to look for composes which are not already in the relations table I think we store the compose name in external_system_id field there. Once a compose is released it doesn't change, so we only need to save new composes.

We could do this as a follow up task if you'd prefer to get this change in first.

Reply:
Yeah, I'll file a follow-up task for this. I want this change to just be "automate what we already run manually", and then we can make improvements to the code after it runs for a while, if nothing shows up in the daily monitoring email.